docs/source/reference/functions.rst

```diff
@@ -1,7 +1,8 @@
-Function Wrappers
+Wrapper Functions
 =================
 
-These functions take an array or a Chainer :class:`Variable` as an argument and return a Chainer :class:`Variable`.
+Wrapper functions are differentiable (backward-able) functions, that take an array or a Chainer :class:`Variable` as an argument and return a Chainer :class:`Variable`, or a tuple of a Chainer :class:`Variable`.
+Wrapper functions may not have learnable parameters when used in models. This is to prevent the functions from changing output due to adjustment of learned parameters during training.
```

Comment:
I meant that a "backward-able" callable should be a `Link` whenever it has parameters (to be trained).  In other words, links that are not added to a target chain of an optimizer will remain unchanged.

The main message would be that we have only to call "wrapper functions" to do differentiable computation, some sentences like
```
Wrapper functions do not have learnable parameters,
so that they do not need to be added to a :class:`Chain` to be trained.
```